.pre-commit-config.yaml

```diff
@@ -117,7 +123,7 @@ repos:
     name: License headers - YAML and Makefiles
     stages: [commit]
     files: (^Makefile|\.(ya?ml|mk))$
-    exclude: ^(pkg/handlers/.+/embedded|examples|charts/cluster-api-runtime-extensions-nutanix/defaultclusterclasses)/.+\.ya?ml|docs/static/helm/index\.yaml$
+    exclude: ^(pkg/handlers/.+/embedded|examples|charts/cluster-api-runtime-extensions-nutanix/defaultclusterclasses)/.+\.ya?ml|docs/static/helm/index\.yaml|charts/cluster-api-runtime-extensions-nutanix/templates/helm-config.yaml$
```

Comment:
This ignores adding the license for this file.

Reply:
Is this needed? The other templates in the chart do have the header

Reply:
It looks like files that we generate through some scripting are ignored. the default clusterclasses do not have them, so i copied that pattern